flink-core/src/main/java/org/apache/flink/api/common/state/StateTtlConfig.java

```diff
@@ -499,6 +496,12 @@ public static class RocksdbCompactFilterCleanupStrategy
          */
         static final Duration DEFAULT_PERIODIC_COMPACTION_TIME = Duration.ofDays(30);
 
+        @Deprecated
```

Comment:
Well, it's better to put `@Deprecated` below the java doc.

Reply:
ok